lib/functions

```diff
@@ -196,3 +196,10 @@ function install_sdc_client {
     mvn install
     popd
 }
+
+# install_unzip() - Download unzip from apt
+function install_unzip {
+    if ! is_package_installed unzip; then
```

Comment:
The install_package function already does a validation[1]

[1] https://github.com/electrocucaracha/vagrant-onap/blob/master/lib/commons#L16